packages/Monorepo/src/Filesystem/Filesystem.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\Monorepo\Filesystem;
+
+use Nette\Utils\FileSystem as NetteFileSystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+final class Filesystem
+{
+    /**
+     * @var string[]
+     */
+    private const EXCLUDED_LOCAL_DIRS = ['packages', 'vendor', '.idea'];
+
+    public function findMergedPackageFiles(string $directory): Finder
+    {
+        return Finder::create()
+            ->files()
+            ->in($directory)
+            ->exclude(self::EXCLUDED_LOCAL_DIRS)
+            // include .gitignore, .travis etc
+            ->ignoreDotFiles(false);
+    }
+
+    /**
+     * - find new files with finder
+     * - copy to new directory
+     */
+    public function copyFinderFilesToDirectory(Finder $finder, string $directory): void
+    {
+        foreach ($finder->getIterator() as $fileInfo) {
+            NetteFileSystem::copy($fileInfo->getRealPath(), $directory . '/' . $fileInfo->getRelativePathname());
```

Comment:
It handles few edge cases like creating existing directory etc. with exceptions instead of silent errors

https://github.com/nette/utils/blob/master/src/Utils/FileSystem.php#L49